tests/tck/steps/test_kill_slow_query_via_different_service.py

```diff
@@ -1,8 +1,7 @@
-# Copyright (c) 2021 vesoft inc. All rights reserved.
+# Copyright (c) 2022 vesoft inc. All rights reserved.
```

Comment:
We only change the year in the header for newly added files.

Reply:
done